src/OpenDebugAD7/MILaunchOptions.cs

```diff
@@ -279,6 +293,47 @@ private static string FormatCommand(JsonCommand command)
             }
         }
 
+        /// <summary>
+        /// To support Windows RunInTerminal's IntegratedTerminal, we need to check and see if the exe arguments contain redirection
+        /// If they don't, we will add redirection to output/input to CON
+        /// </summary>
+        /// <returns></returns>
+        private static string[] TryAddWindowsDebuggeeConsoleRedirection(string[] args)
+        {
+            bool isRedirected = false;
+            foreach (var arg in args)
+            {
+                int index = arg.TrimStart().IndexOf('>');
+                if (index >= 0 && index < 2)
+                {
+                    isRedirected = true;
+                    break;
+                }
+                else
+                {
+                    index = arg.TrimStart().IndexOf('<');
+                    if (index == 0)
+                    {
+                        isRedirected = true;
+                        break;
+                    }
+                }
```

Comment:
1. I might suggest instead just having the set of allowed redirection suffixes prefixes: "<", ">", "1>", "2>"

Reply:
RE: 1. since we only allow it at the beginning, do we want to do indexof each of these?

RE: 2. if the user chose to redirect, do we still want to redirect everything else to console? I was thinking about all or none approach. I think where this might be necessary to allow a different option would be `<` or `0>` for stdin in that if it is specified, then still redirect stdout/err